### PR TITLE
add shop fields to api

### PR DESCRIPTION
**Pull request by PouryaMansouri**

### CODE REVIEW

File: nakhll_market/api.py [hunk 1]
```diff
@@ -679,6 +679,20 @@ def patch(self, request, shop_slug, format=None):
...
+class DeleteShopImage(views.APIView):
+    permission_classes = [permissions.IsAuthenticated, IsShopOwner]
+
+    def get_object(self, shop_slug, user):
+        return get_object_or_404(Shop, Slug=shop_slug)
+
+    def delete(self, request, shop_slug, format=None):
+        user = request.user
+        shop: Shop = self.get_object(shop_slug, user)
+        self.check_object_permissions(request, shop)
```
Comment (likecodingloveproblems) [inline nakhll_market/api.py:691]: @PouryaMansouri Django permission _classes does not check object permissions, only check Model level permissions?

File: nakhll_market/models.py [hunk 2]
```diff
@@ -1178,7 +1188,7 @@ def clean(self):
 
     def _check_product_title_uniqueness_in_shop(self):
         # Check Product Title must be unique in shop
-        if Product.objects.filter(Title = self.Title, FK_Shop_id = self.FK_Shop_id).exist():
+        if Product.objects.filter(Title=self.Title, FK_Shop_id=self.FK_Shop_id).exist():
```
Comment (likecodingloveproblems) [inline nakhll_market/models.py:1191]: have you configured the styling tools mentioned in README?

File: nakhll_market/serializers.py [hunk 3]
```diff
@@ -684,7 +684,8 @@ class Meta:
         model = Shop
         fields = [
             'Title', 'Slug', 'Image', 'image_thumbnail_url',
-            'bank_account', 'social_media', 'Description', 'FK_ShopManager',
+            'bank_account', 'social_media', 'Description', 'FK_ShopManager',"state","BigCity",
+            "City","Location"
```
Comment (likecodingloveproblems) [inline nakhll_market/serializers.py:688]: please, `"` OR `'` NOT BOTH.
```suggestion
            'bank_account', 'social_media', 'Description', 'FK_ShopManager', 'state', 'BigCity',
            'City', 'Location'

**Review by likecodingloveproblems — CHANGES_REQUESTED** [review]
please config styling options

File: nakhll_market/api.py (hunk 1 above)
Comment (PouryaMansouri) [reply nakhll_market/api.py:691]: we check object permission in line 690.

Comment (PouryaMansouri) [reply nakhll_market/api.py:691]: I just follow the code in other views.

Comment (PouryaMansouri) [reply nakhll_market/api.py:691]: I 

> @PouryaMansouri Django permission _classes does not check object permissions, only check Model level permissions?

I think Django permission can check everything.
BUT in prefer to follow the code.

File: nakhll_market/models.py (hunk 2 above)
Comment (PouryaMansouri) [reply nakhll_market/models.py:1191]: > have you configured the styling tools mentioned in README?

I use Pycharm's plugins to reformat the code.
This style was suggested with these plugins. ( Pylint, pep8 and etc ... )

**Comment by PouryaMansouri** [comment]
@likecodingloveproblems please check this request too.

File: nakhll_market/api.py (hunk 1 above)
Comment (likecodingloveproblems) [reply nakhll_market/api.py:691]: :laughing:

File: nakhll_market/models.py (hunk 2 above)
Comment (likecodingloveproblems) [reply nakhll_market/models.py:1191]: OK, please set it up with our style document in README

**Review by likecodingloveproblems — APPROVED** [review]

**Pull request merged**